### DISCUSSION TITLE
I can't figure out how to use useBreakpointValue with remix.

### DISCUSSION THREAD

**Discussion by oiojin831** [chakra-ui/chakra-ui#5525] [Q&A] [answered]
I guess I have to run `useBreakpointValue` when the app is hydrate it.
But not sure how to do it.

Thanks.

**Comment by matthewrose** [comment, marked as answer]
Check out this issue for a possible fix: https://github.com/chakra-ui/chakra-ui/issues/2601

**Reply by oiojin831** [reply]
thank you very much!!
You saved my life :D
